app/controllers/katello/api/v2/organizations_controller.rb

```diff
@@ -72,7 +69,6 @@ def update
     api :POST, '/organizations', N_('Create organization')
     param :name, String, :desc => N_("name"), :required => true
     param :label, String, :desc => N_("unique label")
-    param :description, String, :desc => N_("description")
```

Comment:
Everything works fine for me (updating, creating org, etc) only the top level description param is missing on create in the apidocs.